.github/workflows/node.js.yml

```diff
@@ -29,3 +29,11 @@ jobs:
     - run: npm ci
     - run: npm run build --if-present
     - run: npm test
+    - name: Trigger next workflow
```

Comment:
Will this trigger a "npm" release after every PR/merge to `dev` and `main`, and is that what we want? Is it time for us to split the `node.js` workflow into one workflow for dev that doesn't trigger the semantic-release workflow, and one for main that does? Or do we _want_ to publish nom packages for dev too so we can test with a package before cutting main? If so, does semantic release differentiate between dev/beta releases and "real" releases users should use?

Reply:
I'm still reading up on the semantic-release readme, but does it kind of "magically" take care of the above concerns?

Reply:
Currently yes, but in the publish.yml, there's a conditional that checks if it's to main before doing anything. With that said, I don't think it would be a bad idea to separate the workflows for dev and main, I just didn't want to add risk changing too much. Also, I like the idea of testing this on dev first so I can also remove the conditional so we can do that.

Reply:
👍 that makes a ton of sense